covalent/executor/executor_plugins/dask.py

```diff
@@ -98,9 +108,11 @@ async def run(self, function: Callable, args: List, kwargs: Dict):
 
             await dask_client
 
-        future = dask_client.submit(function, *args, **kwargs)
+        future = dask_client.submit(dask_wrapper, function, args, kwargs)
         app_log.debug("Submitted task to dask")
-        result = await dask_client.gather(future)
+        result, worker_stdout, worker_stderr = await dask_client.gather(future)
+
+        print(worker_stdout, end="")
```

Comment:
```suggestion
        print(worker_stdout, end="", file=sys.stdout)
```
Is `sys.stdout` the default?